test/package.json

```
@@ -0,0 +1,27 @@
+{
+  "name": "@bedrock/redis-test",
+  "version": "1.0.0",
+  "description": "Test suites for bedrock-redis",
+  "type": "module",
+  "scripts": {
+    "test": "node --preserve-symlinks test.js test",
+    "coverage": "c8 --reporter=text --reporter=text-summary npm test",
+    "coverage-ci": "c8 --reporter=lcov npm test",
+    "coverage-report": "c8 report"
+  },
+  "dependencies": {
+    "@bedrock/core": "^6.0.1",
+    "@bedrock/redis": "file:..",
+    "@bedrock/test": "^8.0.5",
+    "c8": "^7.12.0"
+  },
+  "c8": {
+    "excludeNodeModules": false,
+    "include": [
+      "node_modules/@bedrock/redis/**"
+    ],
+    "exclude": [
+      "node_modules/@bedrock/redis/node_modules/**"
+    ]
```

Matching pattern used in other modules:
```suggestion
    ],
    "reporter": [
      "lcov",
      "text-summary",
      "text"
    ]
```

---

Why put the reporters here and in the scripts? Which takes precedent, the ones from the config or the ones specified in the commands?

---

Using `--reporter` will override these defaults.  The theory here was to add defaults that would work for the common "coverage" and "coverage-report" scripts, and then add all `--reporter` options to "coverage-ci" but use `lcovonly`. 
 `lcovonly` omits html output, which is of no use for the CI target.  But I've found it's useful to see the text output in CI output, so had to add all three reporters there.  Having defaults seemed useful for the (rare) times when you might run c8/nyc by hand with some other option but wanted the default reporters.